text/src/autogluon/text/automm/predictor.py

```diff
@@ -71,6 +71,8 @@
 
 logger = logging.getLogger(AUTOMM)
 
+RAY_TUNE_CHECKPOINT = 'ray_tune_checkpoint.ckpt'
```

Comment:
nit: Consider to move it to https://github.com/awslabs/autogluon/blob/master/text/src/autogluon/text/automm/constants.py